src/app/jumio/jumio-success/jumio-success.component.html

```diff
@@ -0,0 +1,4 @@
+<app-banner></app-banner>
+<div>
+  Thank you for verifying your identity.
```

Comment:
Does this appear correctly? The lack of structure makes me think it will look very weird. If we're redirecting immediately do we even want to render anything at all?

Reply:
Maybe copy the Google loader?

Reply:
we redirect users immediately - this was only for testing, so I should remove it.

Is your suggestion to copy the google loader still valid?